aerleon/lib/policy.py

```diff
@@ -2592,7 +2606,7 @@ def p_strings_or_ints(p):
             p[0] = [p[1]]
 
 
-def p_error(p):
+def p_error(p: LexToken):
```

Comment:
Good catch by monkeytype here - the PLY docs consistently refer to the argument to `p_error` as "p" even though it is a LexToken and not a production. Would be more correct as "t".